### PR TITLE
Refactor variables, buttons & text styles

### PR DESCRIPTION
**Pull request by corwinharrell**
Various refactors and updates to variables, buttons, and text styles.
- Add 8 pt vertical rhythm variables
- Introduce relative units (`rem`) for spacing and typography
- Refactor buttons and variables

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Header = props => {
         <BeeLogo />
       </div>
       <div className={`${styles.buttonArea}`}>
-        <Button style="create">Create New Bounty</Button>
+        <Button style="primary">Create New Bounty</Button>
```
Comment (villanuevawill) [inline src/components/Header/index.js]: much better than create :)

**Comment by villanuevawill** [comment]
@corwinharrell looking good. Have you started thinking about font size changes dynamically with screensize?

File: src/components/Header/index.js (hunk 1 above)
Comment (corwinharrell) [reply src/components/Header/index.js]: Yeah we really have no need for a unique "create" btn. Looking to delete that component.

**Comment by corwinharrell** [comment]
@villanuevawill yep! The way I have it set up now, the font-size is defined at the root, and then all font sizes for components are defined in `rem` units relative to the root. We'll introduce media queries that change the root font size for smaller screens, resulting in all of the component font-sizes scaling down proportionally.

**Pull request merged**